hikari/internal/routes.py

```diff
@@ -253,7 +253,7 @@ def compile(
                 + ", ".join(self.valid_formats)
             )
 
-        if "hash" in kwargs and not kwargs["hash"].startswith("a_") and file_format == GIF:
+        if "hash" in kwargs and not str(kwargs["hash"]).startswith("a_") and file_format == GIF:
```

Comment:
Ok so this isn't a fix to the issue as much as it just obfuscates it, really the cdn sticker routes are erroneously referring to pack_id and sticker_id as "hash" leading to this logic being triggered on them when it shouldn't so those should probably be renamed.
Also there's a point here for maybe moving "hash" to a separate optional key-word argument which is explicitly typed as `str` to avoid issues like this in the future

Reply:
I havent had time to fix this, but I did have a different way of fixing thus, but I have been unable to do it just yet. Will update the PR when I get to it.